backend/src/gamer/gamer.types.ts

```diff
@@ -0,0 +1,28 @@
+import { Post } from "../post/post.types";
+import { Game } from "../game/game.types";
+
```

Comment:
Why would you need such a file since we already defined an IGamer interface previously in 'models/Gamer.ts' ?